docs/organizations/managing-people.md

```diff
@@ -15,7 +15,7 @@ To list and manage the members of your organization, open your organization **Se
 
 To become a member of an organization on Codacy you must [sign up to Codacy](../getting-started/codacy-quickstart.md) using your Git provider and follow the instructions to either join an existing organization or add a new one.
 
-To join or add an organization after having complete the signup process, click **Organizations** on the top right-hand menu under your avatar:
+To join or add an organization after completing the signup process, click **Organizations** on the top right-hand menu under your avatar:
```

Comment:
I would never have thought that we would hit this Vale error so fast [after having removed the word `signup` from the accept list](https://github.com/codacy/pulse-user-docs/pull/167#discussion_r1062613135)! :laughing:

Reply:
😄 

```suggestion
To join or add an organization after completing the sign-up process, click **Organizations** on the top right-hand menu under your avatar:
```